src/components/Board.js

```diff
@@ -16,10 +16,71 @@ class Board extends Component {
     };
   }
 
+  componentDidMount() {
+    axios.get('https://inspiration-board.herokuapp.com/boards/ultra_princess/cards')
+      .then((res) => {
+        this.setState({ cards: res.data })
+      })
+      .catch((error) => {
+        alert('hm ur cards didnt load maybe u should refresh lol');
+      });
+  }
+
+  delCard = (id) => {
+    const url = 'https://inspiration-board.herokuapp.com/cards/' + id.toString();
+
+    axios.delete(url)
```

Comment:
You should use complete words for variable and function names. Your jargon may feel obvious to you, but it will often be less so to your coworkers.